pylint/checkers/similar.py

```diff
@@ -93,15 +376,32 @@ def run(self):
     def _compute_sims(self):
         """compute similarities in appended files"""
         no_duplicates = defaultdict(list)
```

Comment:
Can you add a type annotation for `no_duplicates`?

Reply:
@cdce8p i did it but i would like to have your opinion on it.